packages/Group/Button/ButtonGroup.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="btn-group">
+    <button v-for="(item,i) in items" :key="i" v-on:click="handleClicked(item.method,i)" :class="{actived: actived === i}">
+      <i v-if="item.prefix" :class="item.icon"></i>
```

Comment:
why not `pre-icon` and `post-icon` like that we avoid having a combination of 2 required properties.